src/osd/OSD.cc

```diff
@@ -1942,9 +1951,9 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     f->dump_bool("success", success);
     f->dump_int("value", value);
     f->close_section();
-  } else if (command == "dump_objectstore_kv_stats") {
+  } else if (admin_command == "dump_objectstore_kv_stats") {
     store->get_db_statistics(f);
-  } else if (command == "dump_scrubs") {
+  } else if (admin_command == "dump_scrubs") {
```

Comment:
you should also change https://github.com/ceph/ceph/pull/13073/files#diff-fa6c2eba8356ae1442d1bf749beacfdfR1958.